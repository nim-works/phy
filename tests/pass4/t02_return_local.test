(TypeDefs
  (Int 8)
  (ProcTy (Type 0))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 1) (Continuations
    (Continuation (Params)
      (Locals (Type 0))
      (Continue 1 (Copy (Local 0)) (List))
    )
    (Continuation (Params (Type 0))))
  )
)