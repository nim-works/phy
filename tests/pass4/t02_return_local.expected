(TypeDefs
  (Int 8)
  (ProcTy (Type 0))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 1)
    (Locals (Type 0))
    (Continuations
      (Continuation (Params)
        (Locals (Local 0))
        (Continue 1
          (Copy (Local 0))
        )
      )
      (Continuation
        (Params (Type 0))
      )
    )
  )
)
